charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -34,6 +34,8 @@ spec:
         args:
         - --port
         - "8080"
+        - "--profiling={{ .Values.controllerManager.profiling }}"
```

Comment:
for reference, this is because the value is a bool, so you must specify it as a single arg.